File: Android/app/src/main/java/catglo/com/deliverydroid/DeliveryDroidApplication.java
package catglo.com.deliverydroid;

import android.app.Application;
import android.util.Log;
import net.danlew.android.joda.JodaTimeAndroid;
import org.mapsforge.map.layer.cache.TileCache;

import java.util.ArrayList;

public class DeliveryDroidApplication extends Application {
	@Override
	public void onCreate() {
		super.onCreate();
		JodaTimeAndroid.init(this);
	}

    //public ArrayList<TileCache> tileCaches = new ArrayList()
}

File: Android/app/src/main/java/catglo/com/deliverydroid/DeliveryDroidApplication.kt
package catglo.com.deliverydroid

import android.app.Application
import android.util.Log
import net.danlew.android.joda.JodaTimeAndroid
import org.mapsforge.map.layer.cache.TileCache

import java.util.ArrayList

class DeliveryDroidApplication : Application() {

    val tileCaches: List<TileCache> = ArrayList()
    override fun onCreate() {
        super.onCreate()
        JodaTimeAndroid.init(this)
    }

    val notificationChannelId = "my_channel_01"


}